### ISSUE TITLE
Maximized windows is missing decoration

### ISSUE THREAD

**Issue by sneakyevil** [klyushnik/aquarelle#1]
Hello,

There is small issue where maximizing window cause the top decoration of window show as gray (dark theme). 
I tried to mess with the `decoration.svg` and added decoration-maximized-center to add basic rectangle with different color, but I just couldn't get it working same way as the original non maximized window that shows same fade & blocky thing at top-right corner.
So maybe you could perhaps fix this somehow?

**Comment by klyushnik** [comment]
@sneakyevil Hello. Unfortunately, it's a limitation of Aurorae engine - for maximized windows, only the decoration-maximized-center is used, so corners (which have a fade and the blocks graphic) won't be rendered. I thought about this too, but the only solution was to make the graphic a part of decoration-center, which means that it will be distorted in every window, so this solution was a no go from the beginning.

Per documentation ( https://develop.kde.org/docs/plasma/aurorae/ ),

> Maximized windows

>In order to better support maximized windows there exists a special frame SVG called decoration-maximized. In the same way as for the general decoration a version for inactive, opaque, and inactive-opaque can be specified: decoration-maximized, decoration-maximized-inactive, decoration-maximized-opaque and decoration-maximized-opaque-inactive. If not provided, the engine falls back to the normal decoration.

>In all cases, **only the center element will be used**. There is no need to specify borders. Please note that in the case of a window with translucent widgets, the center element will be stretched to the size of the complete window. In order to support Fitts' Law all TitleEdge Settings are set to 0. So the buttons will be directly next to the screen edges.

**Comment by klyushnik** [comment]
@sneakyevil I'll try to fix the gray color in maximized windows, but I don't know if I can add the graphic and the fade to maximized decoration. The problem is that it will stretch on widescreen monitors, not sure how it will behave across different resolutions. The problem is, only decoration-maximized is used, so it will be stretched all the way across. The fade and the graphic is in the corners, so they will always be in the same place.

Per documentation ( https://develop.kde.org/docs/plasma/aurorae/ ):

>In all cases, only the center element will be used. There is no need to specify borders. Please note that in the case of a window with translucent widgets, the center element will be stretched to the size of the complete window. In order to support Fitts' Law all TitleEdge Settings are set to 0. So the buttons will be directly next to the screen edges.

**Comment by sneakyevil** [comment]
Yeah I noticed that it doesn't support any other corner while trying mess with that. I didn't really read everything in that documentation. The main issue is that when it get maximized the color gets to gray color which makes button barely visible. I actually using this on steam deck which has kind of small screen in first place so that's main issue. I already edited myself the decoration.svg with the black color as I said which is still kind of meh with no fade at all. I still think there might be perhaps some way to do some average stuff to properly scale it even when using just center part? Or another way would be to just give guide to user which part of file to edit based on their screen resolution to fit it properly.
